tests/suites/test_suite_pk.data

```diff
@@ -1459,8 +1459,13 @@ pk_import_into_psa_opaque:PSA_KEY_TYPE_ECC_KEY_PAIR(MBEDTLS_TEST_PSA_ECC_ONE_FAM
 Copy from PSA: use wrong parameters
 pk_copy_from_psa_fail:
 
-Copy from PSA: accelerated key only, not available as built-in
-pk_copy_from_psa_builtin_fail:
+Copy from PSA: RSA PKCS1V15 accelerated only, not available as built-in
+depends_on:MBEDTLS_PSA_ACCEL_ALG_RSA_PKCS1V15_SIGN:MBEDTLS_PSA_ACCEL_KEY_TYPE_RSA_KEY_PAIR_BASIC:!MBEDTLS_RSA_C
+pk_copy_from_psa_builtin_fail:PSA_KEY_TYPE_RSA_KEY_PAIR:PSA_ALG_RSA_PKCS1V15_SIGN(PSA_ALG_SHA_256)
+
+Copy from PSA: EC SECP256R1 accelerated only, not available as built-in
+depends_on:MBEDTLS_PSA_ACCEL_ECC_SECP_R1_256:MBEDTLS_PSA_ACCEL_KEY_TYPE_ECC_KEY_PAIR_BASIC:MBEDTLS_PSA_ACCEL_ALG_ECDSA:!MBEDTLS_ECDSA_C:!MBEDTLS_ECP_DP_SECP256R1:MBEDTLS_PSA_ACCEL_ALG_SHA_256
```

Comment:
The CI didn't failed but I suspect that the reason is that this test was never executed, so no disparity was found.
As I commented [above](https://github.com/Mbed-TLS/mbedtls/pull/8774#discussion_r1519896004) I think that trying to use `pk_copy_from_psa_builtin_fail` for EC keys cannot work. In a scenario in which a certain EC key is only available through a driver and not as builtin `mbedtls_pk_copy_from_psa()` uses functions that will all succeed and therefore it will succeed [^1]
Am I missing something?

[^1]: I tried using `component_test_psa_crypto_config_accel_ecc_no_ecp_at_all` and modifying `pk_copy_from_psa_builtin_fail` and its data accordingly) and indeed it succeeds.

Reply:
> In a scenario in which a certain EC key is only available through a driver and not as builtin **mbedtls_pk_copy_from_psa() uses functions that will all succeed** and therefore it will succeed

Oh, I see. I'd missed that part. So indeed we don't need a separate case for ECC, because it's handled by `pk_copy_from_psa_success` test cases.

I prefer the RSA test case the way it is now, can you please keep it and replace the ECC test case by a comment explaining that there's no ECC test case because of what you just wrote?

Reply:
Sure, I can revert https://github.com/Mbed-TLS/mbedtls/pull/8774/commits/7dc2bf2cb0b320e4a43c1978951b1fba02412f0e and add the comment ;)